### ISSUE TITLE
pip install --prefix /tmp easybuild ==> no such option: --prefix

### ISSUE THREAD

**Issue by malcook** [hpcugent/easybuild-framework#1148]
I find the [alternative installation instructions for using pip](http://easybuild.readthedocs.org/en/latest/Installation_Alternative.html#standard-installation-of-latest-release
) to be in error.

Using:
```
$pip --version
pip 1.5.6 from /usr/lib/python2.7/site-packages (python 2.7)
```
I get:
```
$pip install --prefix  /tmp easybuild

Usage:   
  pip install [options] <requirement specifier> ...
  pip install [options] -r <requirements file> ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

no such option: --prefix
```

Rather, the following succeeds

```
pip install --install-option "--prefix=${MODULEPATH_ROOT}" easybuild
Downloading/unpacking easybuild
...
```

**Comment by boegel** [comment]
@malcook: I could have **sworn** I've used `--prefix` with `pip` before, but since it doesn't exist, I'm mistaken.

I also checked whether there's a true equivalent to `--prefix`, but neither `--target` nor `--root` are. With `--target`, the scripts/data file aren't installed in the specified prefix (only Python packages are), while `--root` simply rewrites `/` to whatever you specify.

So, your suggestion of using `--install-option "--prefix=<prefix>"` is indeed the correct one, I'll fix the documentation.

Thanks for reporting this!
